heartbeat/monitors/active/http/http_test.go

```diff
@@ -412,7 +414,7 @@ func TestJsonBody(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
-			server := httptest.NewServer(hbtest.CustomResponseHandler([]byte(tc.responseBody), 200))
+			server := httptest.NewServer(hbtest.CustomResponseHandler([]byte(tc.responseBody), 200, map[string]string{}))
```

Comment:
```suggestion
			server := httptest.NewServer(hbtest.CustomResponseHandler([]byte(tc.responseBody), 200, nil))
```

would be cleaner

Reply:
https://github.com/elastic/beats/pull/28739/commits/af7937235ddb652ec3a2551b958a0aca1525cb2a